examples/ember-data/app/models/album.js

```diff
@@ -1,11 +1,11 @@
-import Model from 'ember-data/model';
-import attr from 'ember-data/attr';
-import { hasMany } from 'ember-data/relationships';
+import Model, { attr, hasMany } from "@ember-data/model";
 
-export default Model.extend({
-  title:    attr('string'),
-  coverUrl: attr('string'),
+export default class Album extends Model {
+  @attr("string") title;
```

Comment:
```suggestion
  @attr title;
```

Reply:
Any particular reason to rely on ED to automatically determine the type vs being explicit?

Reply:
I don't know the exact reason tbh but I know it's the recommendation.